File: Hoops.Core/Entities/Team.cs
using System;
using System.Collections.Generic;

namespace Hoops.Core.Entities
{
    public partial class Team
    {
        public int TeamId { get; set; }
        public int? CompanyId { get; set; }
        public int? SeasonId { get; set; }
        public int? DivisionId { get; set; }
        public int? CoachId { get; set; }
        public int? AssCoachId { get; set; }
        public int? SponsorId { get; set; }
        public string TeamName { get; set; }
        public string TeamColor { get; set; }
        public int TeamColorId { get; set; }
        public string TeamNumber { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string CreatedUser { get; set; }
        public virtual Color Color { get; set; }
        // public virtual Coach Coach { get; set; }
    }
}

File: hoops.ui/src/app/domain/team.ts
export class Team {
    teamId!: number;
    divisionId!: number;
    name?: string;
    teamColorId?: number;
    teamName?: string;
    teamNumber?: string;
    createdDate?: Date;
    createdUser?: string | undefined;
    constructor(
      _teamId? : number,
      _divisionId?: number,
      _name?: string,
      _teamName?: string,
      _teamNumber?: string,
    _teamColorId?: number) {
      this.teamId = _teamId as number;
      this.divisionId = _divisionId as number;
      this.name = _teamName;
      this.teamName = _teamName;
      this.teamNumber = _teamNumber;
      }
}
